site/en/codelabs/openthread-border-router/index.lab.md

```diff
@@ -40,151 +47,42 @@ A Thread Border Router connects a Thread network to other IP-based networks, suc
 
 [OpenThread Border Router](https://openthread.io/guides/border-router) (OTBR) released by Google is an open-source implementation of the Thread Border Router.
 
-### **What you'll build**
+### What you will build
```

Comment:
Can you switch these back?  They can be "you'll"

This is the recommended format and we use it on all the other codelabs